### PR TITLE
Explain message and messageerror for ServiceWorkerContainer.

### PR DESCRIPTION
**Pull request by mfalken**
Addresses #1549

***
<a href="https://pr-preview.s3.amazonaws.com/mfalken/ServiceWorker/pull/1572.html" title="Last updated on Mar 25, 2021, 5:43 AM UTC (2f7989b)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/ServiceWorker/1572/414e78c...mfalken:2f7989b.html" title="Last updated on Mar 25, 2021, 5:43 AM UTC (2f7989b)">Diff</a>

### CODE REVIEW

File: docs/index.bs [hunk 1]
```diff
@@ -803,11 +803,11 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
           </tr>
           <tr>
             <td><dfn attribute for="ServiceWorkerContainer"><code>onmessage</code></dfn></td>
-            <td>{{message!!event}}</td>
+            <td>{{ServiceWorkerContainer/message!!event}}</td>
           </tr>
           <tr>
             <td><dfn attribute for="ServiceWorkerContainer"><code>onmessageerror</code></dfn></td>
-            <td>{{messageerror!!event}}</td>
+            <td>{{ServiceWorkerContainer//messageerror!!event}}</td>
```
Comment (jakearchibald) [inline docs/index.bs:810]:
```suggestion
            <td>{{ServiceWorkerContainer/messageerror!!event}}</td>
```

File: docs/index.bs [hunk 2]
```diff
@@ -872,6 +872,16 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
           <td>{{Event}}</td>
           <td>The [=ServiceWorkerContainer/service worker client=]'s <a>active service worker</a> changes. (See step 9.2 of the <a>Activate</a> algorithm. The <a>skip waiting flag</a> of a [=/service worker=] causes <a lt="activate">activation</a> of the [=/service worker registration=] to occur while [=/service worker clients=] are <a>using</a> the [=/service worker registration=], {{ServiceWorkerContainer/controller|navigator.serviceWorker.controller}} immediately reflects the <a>active worker</a> as the [=/service worker=] that <a>controls</a> the [=/service worker client=].)</td>
         </tr>
+        <tr>
+          <td><dfn event id="service-worker-container-message-event"><code>message</code></dfn></td>
+          <td>{{Event}}</td>
+          <td>The [=ServiceWorkerContainer/service worker client=] receives a message from a [=service worker=]. See {{Client/postMessage(message, options)}}.</td>
```
Comment (jakearchibald) [inline docs/index.bs:878]:
```suggestion
          <td>The [=ServiceWorkerContainer/service worker client=] receives a message from a [=/service worker=]. See {{Client/postMessage(message, options)}}.</td>
```

File: docs/index.bs [hunk 3]
```diff
@@ -872,6 +872,16 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
...
+        </tr>
+        <tr>
+          <td><dfn event id="service-worker-container-messageerror-event"><code>messageerror</code></dfn></td>
+          <td>{{Event}}</td>
+          <td>The [=ServiceWorkerContainer/service worker client=] is sent a message that cannot be deserialized from a [=service worker=]. See {{Client/postMessage(message, options)}}.</td>
```
Comment (jakearchibald) [inline docs/index.bs:883]:
```suggestion
          <td>The [=ServiceWorkerContainer/service worker client=] is sent a message that cannot be deserialized from a [=/service worker=]. See {{Client/postMessage(message, options)}}.</td>
```

**Review by jakearchibald — APPROVED** [review]
LGTM with the changes suggested

**Comment by mfalken** [comment]
Thanks!

**Pull request merged**